packages/perseus/src/widgets/radio/choice.jsx

```diff
@@ -168,6 +185,26 @@ function Choice(props: ChoicePropsWithForwardRef): React.Node {
                     opacity: showDimmed ? 0.5 : 1.0,
                 }}
             >
+                <div className="perseus-sr-only">
+                    <input
+                        type={multipleSelect ? "checkbox" : "radio"}
+                        id={choiceId}
+                        name={questionId}
```

Comment:
Are we sure we need both `questionId` and `choiceId`?

Reply:
Will try putting the input inside the label and see if that works. If so, will remove all the choiceIds.

Reply:
Putting the input inside the label seemed to work the same as this way, so removed choiceId and all the places that used it and moved input inside label.